tests/examples/HHClientLinter/invalid_null_check.hack.expect

```diff
@@ -1,7 +1,7 @@
 [
     {
-        "blame": "cannot_be_null is null",
-        "blame_pretty": "cannot_be_null is null",
+        "blame": "$cannot_be_null is null",
+        "blame_pretty": "$cannot_be_null is null",
```

Comment:
on another PR, I mentioned we should add ranges here; we should add error codes too. Again shouldn't be this PR, because it'll need a lot of things updating.